dev/com.ibm.ws.jsf.2.3_fat/test-applications/CDICommon/src/com/ibm/ws/jsf23/fat/cdi/common/managed/CustomELResolver.java

```diff
@@ -85,9 +93,17 @@ public Object getValue(ELContext context, Object base, Object property) {
                 else
                     outcome += _methodBean.getData();
 
-                ((TestCustomBean) base).setData(outcome);
+                if (className40Found) {
+                    ((com.ibm.ws.jsf23.fat.cdi.common.beans.faces40.TestCustomBean) base).setData(outcome);
+                } else {
+                    ((TestCustomBean) base).setData(outcome);
+                }
```

Comment:
Small style thing: you have curly brackets for these if statements but not above.

Reply:
I fixed it thanks!